.gitignore

```diff
@@ -15,3 +15,5 @@ tmp/
 .ruby-version
 
 .bundle/
+
+.idea/
```

Comment:
I'm OK with this....but trying to understand what created this. Was it your IDE?

Reply:
OH YES jetbrains products, no matter who it is, loves to add some `ideas`

Reply:
I see! While still fine with this (:grin:), I'm wondering if it would be a better candidate for your ~/.gitignore file?

Reply:
OH OH gotcha, yeah sure down with that, just wasn't sure if anyone else had ever... because if they had... it might make sense to... but agreed, kinda specific, has nothing to do with codebase, will switch it back